config/config.go

```diff
@@ -132,6 +129,7 @@ func (c *Config) Prepare() error {
 	if err := os.MkdirAll(c.Path, os.FileMode(0755)); err != nil {
 		return err
 	}
+	// TODO: find a way to not hardcode "config" and "data" folder from tendermint
```

Comment:
Why? What is the value of changing the tendermint dir layout?

Reply:
I just would like to get the value from tendermint config instead of hardcoded it in our codebase.
I don't want to change it or whatever, just don't want to define it and use the default tendermint one.

Reply:
I found a way to do it :)